apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/staticelement/PageStaticController.java

```diff
@@ -0,0 +1,87 @@
+package gov.cdc.nbs.questionbank.page.content.staticelement;
+
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import gov.cdc.nbs.authentication.UserDetailsProvider;
+import gov.cdc.nbs.questionbank.page.content.staticelement.request.PageStaticRequests;
+import gov.cdc.nbs.questionbank.page.content.staticelement.response.AddStaticResponse;
+import lombok.extern.slf4j.Slf4j;
+
+
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v1/pages/{page}/static-element/")
+@PreAuthorize("hasAuthority('LDFADMINISTRATION-SYSTEM')")
+public class PageStaticController {
+    private final UserDetailsProvider userDetailsProvider;
+    private final PageStaticCreator pageStaticCreator;
+
+    public PageStaticController(
+            final UserDetailsProvider userDetailsProvider,
+            final PageStaticCreator pageStaticCreator) {
+        this.userDetailsProvider = userDetailsProvider;
+        this.pageStaticCreator = pageStaticCreator;
+    }
+
+
+    @PostMapping("/line-separator")
+    public AddStaticResponse addStaticLineSeparator(
+            @PathVariable("page") Long pageId,
+            @RequestBody PageStaticRequests.AddStaticElementDefaultRequest request) {
+        log.debug("Received request to add line separator in page");
+        Long userId = userDetailsProvider.getCurrentUserDetails().getId();
```

Comment:
polish:  The UserDetails are already being loaded to handle the pre-authorize, you can get the current value by adding the`@ApiIgnore @AuthenticationPrincipal final NbsUserDetails details` parameter to the method.  The `@ApiIgnore` annotation is added to prevent it from showing up in Swagger.